dev-libs/uchardet/uchardet-0.0.5-r1.ebuild

```diff
@@ -6,8 +6,8 @@ EAPI=5
 
 inherit cmake-utils
 
-DESCRIPTION="An encoding detector library ported from Mozilla"
-HOMEPAGE="https://github.com/BYVoid/uchardet"
+DESCRIPTION="An encoding detector library"
+HOMEPAGE="https://github.com/BYVoid/uchardet https://www.freedesktop.org/wiki/Software/uchardet/"
```

Comment:
Yes, because this release isn't available from fdo repository. And fdo repository doesn't point back to github one.